lib/provider-retrieval-result-stats.js

```diff
@@ -21,6 +21,9 @@ export const build = committees => {
   for (const c of committees) {
     // IMPORTANT: include minority results in the calculation
     for (const m of c.measurements) {
+      // Ignore measurements reporting IPNI service outage
+      if (m.retrievalResult.match(/^IPNI_ERROR_5\d\d$/)) continue
```

Comment:
```suggestion
      if (m.retrievalResult.startsWith('IPNI_ERROR_5')) continue
```

Simpler, but no big deal

Reply:
Also applies below